lib/checkPullRequestJob.js

```diff
@@ -0,0 +1,58 @@
+const {SERVER_JOBS_ADMIN} = require('../userWhitelist.json');
+/**
+ * @param {string} filename
+ */
+const isJobFile = (filename) => {
```

Comment:
What if a job file is being modified but no new job is introduced? There can be a case when an issue is being fixed or some variables are being renamed. We only need the test procedure for new jobs.

Further, we need to have a check which ensures that whenever a new job is added, the name of the job is added to jobs_registry file.

Reply:
We may not be able to check the exact content of the job_registry file. What we can do is to ensure that a new job file is accompanied with a change in the job_registry.

Reply:
> We may not be able to check the exact content of the job_registry file. What we can do is to ensure that a new job file is accompanied with a change in the job_registry.

I've been able to do this.